src/test/java/edu/hawaii/its/api/controller/RestExceptionHandlerTest.java

```diff
@@ -0,0 +1,134 @@
+package edu.hawaii.its.api.controller;
+
+import edu.hawaii.its.api.configuration.SpringBootWebApplication;
+import edu.hawaii.its.api.exception.AccessDeniedException;
+import edu.hawaii.its.api.exception.UhMemberNotFoundException;
+import edu.hawaii.its.api.service.*;
```

Comment:
Make sure to remove any * imports, thanks.

Reply:
Thanks for letting me know